### PR TITLE
Vode pie guardbuddy

### PR DESCRIPTION
**Pull request by vode-code**
[FEATURE]
## About the PR 
This PR adds a currently unobtainable guardbuddy tool module that allows guardbuddies to throw pies.
The pies are projectiles that apply standard thrown pie effects on hit.
Although the module is currently using the default tool module item sprite, it is fully operational and can be used as an admin-only item.

EDIT as status update.
The tool module now is able to be bought from Geoff Honkington, and now has a(low effort) sprite.
To go along with the tool module, there is now a "Clownbuddy", which has the clown buddy costume sprite, along with the pie launcher tool module.

### CODE REVIEW

**Comment by Carbadox** [comment]
Maybe give it to Geoff Honkington so that he can sell it?

**Comment by ZeWaka** [comment]
title and description were confusing...
to clarify, this PR adds a currently unobtainable guardbuddy tool module that launches a pie projectile.

**Comment by Tarmunora** [comment]
Should have implementation (clown merchant sounds good) and a spawnable buddy for it. Sprite would be nice to have as well.

**Comment by vode-code** [comment]
I added it to Geoff, don't have a custom buddy or sprite yet.

File: code/datums/commodity.dm [hunk 1]
```diff
@@ -1243,6 +1243,15 @@
...
 	lowerfluc = -50
 
+/datum/commodity/junk/pie_launcher
+	comname = "Pie Tool Module"
+	comtype = /obj/item/device/guardbot_tool/pie_launcher
+	desc = "A tool module compatible with guardbuddies. Are tool modules supposed to have cream on them?"
+	price = PAY_DOCTORATE/2
+	baseprice = PAY_DOCTORATE/2
+	upperfluc = PAY_DOCTORATE/6
+	lowerfluc = -PAY_DOCTORATE/6
```
Comment (Tarmunora) [inline code/datums/commodity.dm:1253]: why base the price off doctorate instead of staffie or clown?

Comment (vode-code) [reply code/datums/commodity.dm:1253]: because it's a Guardbuddy module, and RD messes with guardbuddies, not clown.

**Comment by vode-code** [comment]
Sprite, Buddy, and method to obtain are all complete.

**Comment by vode-code** [comment]
However, the sprite isn't very refined, and there is room for improvement.

**Pull request merged**